test/functionalTest/cases/v2/retrieve_entry_point.test

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
The subdirectories within cases/ follow the pattern: NNN_XXXX where NNN is the number of the issue which functionality/fix is being tested and XXX is a descriptive string. In this case I suggest:

956_list_all_entry_points